### DISCUSSION TITLE
Say Hi to the CSS Battle community.

### DISCUSSION THREAD

**Discussion by niawjunior** [cssbattle/forum#358] [Show and tell]
Say Hi to the CSS Battle community.

Hi folks, 
I want to begin by expressing my gratitude to the CSSBattle platform for providing an interactive website for learning CSS. Personally, I've struggled with CSS and have been working hard to improve my skills. That's when I discovered the CSSBattle website. My friends and I really appreciate this platform, and we frequently use it to practice our CSS skills. In the last two months, I started creating a website that showcases our CSS solutions from CSSBattle to track our progress.

Here is my website, and please feel free to contribute or create your PR to showcase your CSS solutions.

Many thanks to the CSSBattle team!

github: https://github.com/peuan/snippetui

website: https://css-battle-eta.vercel.app/
<img width="1436" alt="image" src="https://github.com/cssbattle/forum/assets/17493195/a1f5d966-f02e-4c16-866c-18d93270f13f">
<img width="1425" alt="image" src="https://github.com/cssbattle/forum/assets/17493195/fc66f1c8-c6b0-4116-be60-1209f5c183d4">
<img width="1438" alt="image" src="https://github.com/cssbattle/forum/assets/17493195/40b576e1-c218-48cc-a5fb-be8194dda7b3">

**Comment by chinchang** [comment]
Thanks for the nice works @niawjunior . Really happy that CSSBattle has been helpful! ❤️ 
Great work on the SnipperUI website! Way to go!
